src/node_launcher.star

```diff
@@ -16,65 +16,72 @@ PUBLIC_IP = "127.0.0.1"
 
 utils = import_module("./utils.star")
 
-def launch(plan, genesis, image, node_count, ephemeral_ports, min_cpu, min_memory, vmId, dont_start_subnets, custom_subnet_vm_path, custom_subnet_vm_url):
+
+def launch(plan, genesis, image, node_count, ephemeral_ports, min_cpu, min_memory, vmId, dont_start_subnets,
+           custom_subnet_vm_path, custom_subnet_vm_url):
     bootstrap_ips = []
     bootstrap_ids = []
     nodes = []
     launch_commands = []
 
     services = {}
     plan.print("Creating all the avalanche containers paralllely")
-    for index in range (0, node_count):
+    for index in range(0, node_count):
         node_name = NODE_NAME_PREFIX + str(index)
 
-        node_data_dirpath =  ABS_DATA_DIRPATH + node_name + "/"
+        node_data_dirpath = ABS_DATA_DIRPATH + node_name + "/"
         node_config_filepath = node_data_dirpath + "config.json"
 
         launch_node_cmd = [
             "nohup",
             "/avalanchego/build/" + EXECUTABLE_PATH,
-            "--genesis=/tmp/data/genesis.json", 
+            "--genesis-file=/tmp/data/genesis.json",
             "--data-dir=" + node_data_dirpath,
             "--config-file=" + node_config_filepath,
             "--http-host=0.0.0.0",
             "--staking-port=" + str(STAKING_PORT_NUM),
-            "--http-port="+ str(RPC_PORT_NUM),
-            "--log-dir=/tmp/"
+            "--http-port=" + str(RPC_PORT_NUM),
+            "--log-dir=/tmp/",
+            "--network-health-min-conn-peers=" + str(node_count - 1),
```

Comment:
I've tried to set this to get the health check to pass in a 1-node setup, but to no avail. It merely changed the health check error from:

"error": "network layer is unhealthy reason: not connected to a minimum of 1 peer(s) only 0, no
messages received from network, no messages sent to network",

to

"error": "network layer is unhealthy reason: no messages received from network, no messages sent to
network",

Reply:
Setup will fail if nodes < 2.